codespell_lib/data/dictionary.txt

```diff
@@ -1939,14 +1962,18 @@ Champange->Champagne
 chaned->changed, chained,
 chaneged->changed
 chanel->channel
+chanels->channels
+chang->change
```

Comment:
i bet will get a lot of false positives with this one

Reply:
removed